src/routes/notification.route.js

```diff
@@ -87,4 +87,45 @@ router.get('/:id', verifyToken.headerToken, notificationController.getNotificati
  *            type: string
  */
 router.patch('/changePreference', Validate.userPreference(), isValid, verifyToken.headerToken, notificationController.changePreference);
+/**
+ * @swagger
+ *
+ * /notifications:
+ *    patch:
+ *      summary: Mark all notifications as read
+ *      tags: [Notification]
+ *      parameters:
+ *       - name: token
+ *         in: header
+ *         description: Check authentication
+ *         required: true
+ *         type: string
+ *      responses:
+ *        "200":
+ *          description: Successfully marked all notifications as read.
+ */
+router.patch('/read', verifyToken.headerToken, notificationController.markAllNotificationsAsRead);
```

Comment:
@VedaDom Kindly change this to PUT or PATCH /notifications and then pass the read or unread property in the body as either false or true